bundles/org.openhab.binding.upnpcontrol/src/main/java/org/openhab/binding/upnpcontrol/internal/handler/UpnpServerHandler.java

```diff
@@ -165,8 +165,8 @@ public void dispose() {
     protected void initJob() {
         synchronized (jobLock) {
             if (!upnpIOService.isRegistered(this)) {
-                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.COMMUNICATION_ERROR,
-                        "UPnP device with UDN " + getUDN() + " not yet registered");
+                String msg = String.format("@text/offline.device-not-registed [ \"%s\" ]", getUDN());
```

Comment:
Doesn't match actual key:
```suggestion
                String msg = String.format("@text/offline.device-not-registered [ \"%s\" ]", getUDN());
```

Reply:
Many thanks and fixed. There was a second place with the mismatch for the same key.